sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -103,6 +109,7 @@ def test_ndframe_tests(self):
             'pandas.core.generic.NDFrame.get': ['*'],
         },
         not_implemented_ok={
```

Comment:
Is the distinction here that there is a list of operations that we 100% won't implement, and then there are the operations that are just not-yet-implemented but could be?

Reply:
yes, that was the original intent. Order-sensitive ops are an example of 'Won't implement', although there is a proposal https://s.apache.org/order-sensitive-dataframe-operations possibly change that. The 'Not implemented' ops raise an exception that says, we didn't get around to support it but let us know if you care. See: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/dataframe/frame_base.py#L426-L442